### PR TITLE
Changed streaming keep alive test to use byteman (CASSANDRA-12929)

### PR DESCRIPTION
**Pull request by pauloricardomg**

### CODE REVIEW

File: bootstrap_test.py [hunk 1]
```diff
@@ -148,23 +148,28 @@ def simple_bootstrap_test_small_keepalive_period(self):
...
+        node1.import_config_files()
+
         cluster.start(wait_other_notice=True)
 
         # Create more than one sstable larger than 1MB
-        node1.stress(['write', 'n=50K', '-rate', 'threads=8', '-schema',
-                      'compaction(strategy=SizeTieredCompactionStrategy, enabled=false)'])
-        cluster.flush()
-        node1.stress(['write', 'n=50K', '-rate', 'threads=8', '-schema',
+        node1.stress(['write', 'n=1K', '-rate', 'threads=8', '-schema',
```
Comment (ptnapoleon) [inline bootstrap_test.py]: This is a big drop in row count. Is it going to be enough?

File: bootstrap_test.py [hunk 2]
```diff
@@ -148,23 +148,28 @@ def simple_bootstrap_test_small_keepalive_period(self):
...
                       'compaction(strategy=SizeTieredCompactionStrategy, enabled=false)'])
         cluster.flush()
-        self.assertGreater(node1.get_sstables("keyspace1", "standard1"), 1)
+
+        debug("Submitting byteman script to {} to".format(node1.name))
+        # Sleep longer than streaming_socket_timeout_in_ms to make sure the node will not be killed
+        node1.byteman_submit(['./byteman/stream_5s_sleep.btm'])
```
Comment (ptnapoleon) [inline bootstrap_test.py]: I don't see this script. Did you mean to include it?

File: bootstrap_test.py (hunk 1 above)
Comment (pauloricardomg) [inline bootstrap_test.py]: yeah, with the byteman script which will sleep on streaming this is just required to create the keyspace and 1 sstable so a few rows is sufficient.

File: bootstrap_test.py (hunk 2 above)
Comment (pauloricardomg) [inline bootstrap_test.py]: oops, forgot to include, nice catch.

Comment (pauloricardomg) [inline bootstrap_test.py]: updated the patch with script

**Comment by ptnapoleon** [comment]
+1

**Pull request merged**